Informatics II/000/Task 3/Patertuck Solution/Ex00_patertuck.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <limits.h>
+
+int main() {
+
+    int pos_largest, pos_second;
+    int largest_number = INT_MIN;
+    int second_largest_number = INT_MIN;
+    int A[] = {4, 3, 11, 133, 9, 8, 312};
+
+    for (int i = 0; i < sizeof(A)/sizeof(A[0]); i++ ) {
```

Comment:
Would put the length (`sizeof(A)/sizeof(A[0])`) into a seperate int above, because like this every time it iterates over the loop, it recalculates the length.

Reply:
ahhh, makes sense